setup.sh

```diff
@@ -0,0 +1,295 @@
+        SqlRoot="root"
+    fi
+    logprint "Creating sql database and setting permission. You now need to enter the password of the root sql user twice"
+    mysql -u $SqlRoot -p < "$InstallFilesFolder/create.sql"
+    mysql -u $SqlRoot -p --execute="GRANT ALL ON $SqlDb.* TO $SqlUser@localhost IDENTIFIED BY '$SqlPwd'; GRANT FILE ON *.* TO $SqlUser@localhost IDENTIFIED BY '$SqlPwd'; FLUSH PRIVILEGES"
+
+    read -p "Enter an admin user for the MailWatch web interface: " MWAdmin
+    read -p "Enter password for the admin: " MWAdminPwd
+    logprint "Create MailWatch web gui admin"
+    mysql -u $SqlUser -p$SqlPwd $SqlDb --execute="REPLACE INTO users SET username = '$MWAdmin', password = MD5('$MWAdminPwd'), fullname = 'Admin', type = 'A';"
+else
+    echo "You have to create the database yourself!"
+    EndNotice="$EndNotice \n * create the database, a sql user with access to the db and following properties user: $SqlUser; password: $SqlPwd; db: $SqlDb; host: $SqlHost"
+    EndNotice="$EndNotice \n * create an admin account for the web gui"
+fi
+
+#copy web files
+logprint "Moving MailWatch web files to new folder and setting permissions"
+mv "$InstallFilesFolder/mailscanner/" $WebFolder
+chown root:mtagroup $WebFolder/images
+chmod ug+rwx $WebFolder/images
+chown root:mtagroup $WebFolder/images/cache
+chmod ug+rwx $WebFolder/images/cache
+chown root:mtagroup $WebFolder/temp
+chmod g+rw $WebFolder/temp
+
+#test existing webserver
+if ( type "httpd" > /dev/null 2>&1 ) || ( type "apache2" > /dev/null 2>&1 ); then
+    WebServer="apache"
+    logprint "Detected installed web server apache. We will use it for MailWatch"
+elif ( type "nginx" > /dev/null 2>&1 ); then
+    WebServer="nginx"
+    logprint "Detected installed web server nginx. We will use it for MailWatch"
+else
+    echo "We're unable to find your webserver.  We support Apache and Nginx";echo;
+    echo "Do you wish me to install a webserver?"
+    echo "1 - Apache"
+    echo "2 - Nginx"
+    echo "n - do not install or configure"
+    echo;
+    read -r -p "Select Webserver: " response
+    if [[ $response =~ ^([nN][oO])$ ]]; then
+        #do not install or configure webserver
+        WebServer="skip"
+    elif [ $response == 1 ]; then
+        #Apache
+        logprint "Installing apache"
+        if [ $PM == "yum" ];
+            $PM install httpd
+            Webuser="apache"
+        else
+            $PM install apache2
+            Webuser="www-data"
+        fi
+        WebServer="apache"
+    elif [ $response == 2 ]; then
+        #Nginx
+        logprint "Installing nginx"
+        if [ $PM == "yum" ];
+            $PM install nginx
+            Webuser="nginx"
+        else
+            $PM install nginx
+            Webuser="www-data"
+        fi
+        WebServer="nginx"
+    else
+        WebServer="skip"
+    fi
+fi
+
+read -p "MailWatch requires the php packages php5 php5-gd and php5-mysqlnd. Do you want to install them if missing?(y/n)[y]: " installPhp
```

Comment:
Should probably check is PHP is already installed, and what version.  

On debian based systems the below will install php5 regardless of any other versions of php.

On RH systems, only 1 version of PHP can be installed by yum - and packages are defined as `php php-gd` etc default version is v5.